packages/wallet/src/redux/protocols/consensus-update/__tests__/reducer.test.ts

```diff
@@ -86,3 +197,23 @@ function itTransitionsTo(
     expect(result.protocolState.type).toEqual(type);
   });
 }
+
+function itSendsMessage(
```

Comment:
I think there's already a helper that does this, that's used in the advance channel tests: `expectCommitmentsSent`